libvcx/src/messages/get_message.rs

```diff
@@ -539,55 +540,56 @@ mod tests {
     }
 
     #[cfg(feature = "agency_pool_tests")]
-    #[cfg(feature = "to_restore")] // todo: failing on "invalid credential preview"
     #[test]
     fn test_download_messages() {
         let _setup = SetupLibraryAgencyV2::init();
 
         let institution_did = settings::get_config_value(settings::CONFIG_INSTITUTION_DID).unwrap();
-        let (_faber, alice) = ::connection::tests::create_connected_connections();
-
-        let (_, cred_def_handle) = ::credential_def::tests::create_cred_def_real(false);
-
-        let credential_data = r#"{"address1": ["123 Main St"], "address2": ["Suite 3"], "city": ["Draper"], "state": ["UT"], "zip": ["84000"]}"#;
-        let credential_offer = ::issuer_credential::issuer_credential_create(cred_def_handle,
-                                                                             "1".to_string(),
-                                                                             institution_did.clone(),
-                                                                             "credential_name".to_string(),
-                                                                             credential_data.to_owned(),
-                                                                             1).unwrap();
+        let (alice_to_faber, faber_to_alice) = ::connection::tests::create_connected_connections();
```

Comment:
Maybe move this to [this PR](https://github.com/AbsaOSS/libvcx/pull/96), to avoid conflicts